### DISCUSSION TITLE
Cant log in ?

### DISCUSSION THREAD

**Discussion by akromdoo** [YetiForceCompany/YetiForceCRM#15113] [General] [answered]
Hi Yesturday all worked ok today i can log in my yetiforce. i dident change any credentials.

Whan i try to log in it only refreshes the page. If i change the user or password with non existing i get mesage wrong username or password.

Checked my file permisions and all ok. Dont know what to do now?

I have 6.2 version

**Comment by rskrzypczak** [comment, marked as answer]
Someone had to change something :).
However, try emptying the cookies first.
![image](https://user-images.githubusercontent.com/10221999/131100345-ec04c085-c3e0-43d6-9d03-09a4edf25e00.png)

**Reply by akromdoo** [reply]
Totaly forgot for cookies :) after that it worked.

Thanks
